libcxx/include/type_traits

```diff
@@ -4843,7 +4843,7 @@ using _IsCharLikeType = _And<is_standard_layout<_CharT>, is_trivial<_CharT> >;
 
 _LIBCUDACXX_END_NAMESPACE_STD
 
-#if _LIBCUDACXX_STD_VER > 14
+#if _LIBCUDACXX_STD_VER > 11
```

Comment:
Does the corresponding endif need to be updated here?

Reply:
Done.